Apps/Sandcastle/gallery/Clipping Planes.html

```diff
@@ -0,0 +1,191 @@
+    <meta name="cesium-sandcastle-labels" content="Showcases, 3D Tiles">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+    #toolbar {
+        background: rgba(42, 42, 42, 0.8);
+        padding: 4px;
+        border-radius: 4px;
+    }
+    #toolbar input {
+        vertical-align: middle;
+        padding-top: 2px;
+        padding-bottom: 2px;
+    }
+    #toolbar .header {
+        font-weight: bold;
+    }
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar">
+    <table><tbody>
+    <tr>
+        <td>Type</td>
+        <td><select data-bind="options: exampleTypes, value: exampleType"></select></td>
+    </tr>
+    <tr>
+        <td>x</td>
+        <td>
+            <input type="range" min="-100" max="100" step="1" data-bind="value: xOffset, valueUpdate: 'input'">
+            <input type="text" size="2" data-bind="value: xOffset">
+        </td>
+    </tr>
+    <tr>
+        <td>y</td>
+        <td>
+            <input type="range" min="-100" max="100" step="1" data-bind="value: yOffset, valueUpdate: 'input'">
+            <input type="text" size="2" data-bind="value: yOffset">
+        </td>
+    </tr>
+    <tr>
+        <td>z</td>
+        <td>
+            <input type="range" min="-100" max="100" step="1" data-bind="value: zOffset, valueUpdate: 'input'">
+            <input type="text" size="2" data-bind="value: zOffset">
+        </td>
+    </tr>
+    </tbody></table>
+</div>
+
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+var defaultClippingPlanes = [
+    new Cesium.Plane(new Cesium.Cartesian3(-1, 0, 0), 0.0),
+    new Cesium.Plane(new Cesium.Cartesian3(0, -1, 0), 100.0),
+    new Cesium.Plane(new Cesium.Cartesian3(0, 0, 1), 100.0)
+];
+
+function loadTileset (url) {
+    reset();
+
+    var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+        url : url
+    }));
+
+    tileset.readyPromise.then(function() {
+        var boundingSphere = tileset.boundingSphere;
+        viewer.camera.viewBoundingSphere(boundingSphere, new Cesium.HeadingPitchRange(0.5, -0.2, boundingSphere.radius * 4.0));
+        viewer.camera.lookAtTransform(Cesium.Matrix4.IDENTITY);
+    }).otherwise(function(error) {
+        throw(error);
+    });
+
+    tileset.clippingPlanes = defaultClippingPlanes;
+    return tileset.clippingPlanes;
+}
+
+function loadModel (url) {
```

Comment:
It is an awkward design to have the `load` functions return the clipping planes array.

In `getObservable`, perhaps create a `new` array and pass it to the load function.  Basically `defaultClippingPlanes` becomes a local.